### PR TITLE
Add secure Jenkins management

### PR DESCRIPTION
**Pull request by jecisc**
Please review since I don't know Jenkins well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,21 @@ zipArtifact := pillar lastSuccessfulBuild runs anyOne artifacts detect: [ :artif
...
+```smalltalk
+| server job zipArtifact | 
+
+server := JenkinsServer name: 'aServerName' url: ('https://yourURL' asUrl) username: 'x.y@something.com'; token: 'aToken'; yourself.
+
+"Your username should be the email address of your Jenkins account.
+You can find your token in the settings of your account.
+Account -> Configure -> API Token"
+
+job := server jobs detect: [ :job | job name = 'aJobName' ].
```
Comment (DamienCassou) [inline README.md:36]: why do you add this?

File: src/Jenkins-Core.package/JenkinsServer.class/README.md [hunk 2]
```diff
@@ -2,4 +2,10 @@ I represent a Jenkins server.
 
 A Jenkins server (http://jenkins-ci.org/) supports continuous integration through jobs.
 
-JenkinsServer name: 'Pharo contributions' url: 'https://ci.inria.fr/pharo-contribution' asUrl
\ No newline at end of file
+JenkinsServer name: 'Pharo contributions' url: 'https://ci.inria.fr/pharo-contribution' asUrl
+
+You have also the possibility to use me with a Jenkins protected by a password. In order to do that you need a username for the jenkins that should be your email. You also need a token that you can fin into the configuration of your Jenkins account.
```
Comment (DamienCassou) [inline src/Jenkins-Core.package/JenkinsServer.class/README.md]: "fin" -> "find"

File: src/Jenkins-Core.package/JenkinsServer.class/instance/token..st [hunk 3]
```diff
@@ -0,0 +1,3 @@
+accessing
+token: anObject
```
Comment (DamienCassou) [inline src/Jenkins-Core.package/JenkinsServer.class/instance/token..st]: `aString`

File: src/Jenkins-Core.package/JenkinsServer.class/instance/username..st [hunk 4]
```diff
@@ -0,0 +1,3 @@
+accessing
+username: anObject
```
Comment (DamienCassou) [inline src/Jenkins-Core.package/JenkinsServer.class/instance/username..st]: `aString`

File: README.md (hunk 1 above)
Comment (jecisc) [inline README.md:36]: I just took the same code as as before (to get the Pillar.zip) to give an example of how to get a zip file on a secure server.

**Comment by jecisc** [comment]
@DamienCassou New commit. 

After the merge, if I have some time, do you want that I change PharoLauncher to use this project instead of the PharoLauncher implementation ?

**Comment by DamienCassou** [comment]
Please squash into 1 commit and I will merge it

**Comment by DamienCassou** [comment]
Please adapt the current launcher to use this project if you have time. Thanks

**Comment by jecisc** [comment]
@DamienCassou Squash done!

**Comment by DamienCassou** [comment]
Thanks

**Pull request merged**